src/c/profile/multithread/multithread.c

```diff
@@ -41,6 +46,11 @@ void uxr_lock(
 void uxr_unlock(
         uxrMutex* mutex)
 {
+    if (mutex == NULL)
+    {
+        return;
+    }
+
```

Comment:
Remove

Reply:
Removed, but added this check for stream mutex:
https://github.com/eProsima/Micro-XRCE-DDS-Client/blob/d9e34f9b4c02fa741b6a45b242613807a3da3d41/include/uxr/client/profile/multithread/multithread.h#L97-L109

I think it is necessary, since `uxr_get_stream_mutex_from_id` can return a NULL mutex pointer:
https://github.com/eProsima/Micro-XRCE-DDS-Client/blob/d9e34f9b4c02fa741b6a45b242613807a3da3d41/src/c/profile/multithread/multithread.c#L64